test/unit/selector.js

```diff
@@ -1326,3 +1326,19 @@ test("select() with pre-compiled function", function() {
 		equal( Sizzle.select( compiled, Sizzle( "#first")[0] ).length, 0, "Should not match with different context" );
 	});
 });
+
+test("Sizzle supports shadow DOM nodes as root", function() {
+	if ( !document.documentElement.createShadowRoot ) {
+		ok( true, "Shadow DOM not supported in this browser" );
+		return;
```

Comment:
We should probably start using the same QUnit syntax in Sizzle at some point. We'll use skip here when that happens.